docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -35,30 +35,26 @@ If the application resumes data successfully:
     * `SubscribeButton`
     * `SubscibeVehicleData`
 
-If the application does not resume data successfully:
+If the application does NOT resume data successfully:
 
   * SDL will provide `OnAppRegistered` with `resumeVrGrammars`:`false` or no resume parameter at all.
   * SDL cleans up all previously stored application data for the application that failed to resume. The HMI must also clean up previously compiled `VRGrammars` for the application.
   * The application will send new data to start SDL operations. In this event, SDL and the HMI should restart the cycle of collecting application data for resumption.
 
-!!! MUST
+_**HMI must:**_
```

Comment:
This section should use the 

```
!!! MUST
Content
!!!
```

format as specified in the [SDL Markdown Spec](https://github.com/smartdevicelink/sdl_markdown_spec).